orbs/account.go

```diff
@@ -56,3 +57,21 @@ func AddressValidate(address string) error {
 func BytesToAddress(rawAddress []byte) string {
 	return encoding.EncodeHex(rawAddress)
 }
+
+const (
+	SHA256_HASH_SIZE_BYTES = 32
+	CLIENT_ADDRESS_SIZE_BYTES    = 20
+	CLIENT_ADDRESS_SHA256_OFFSET = SHA256_HASH_SIZE_BYTES - CLIENT_ADDRESS_SIZE_BYTES
+)
+
+// https://github.com/orbs-network/orbs-network-go/commit/681e1383bbfaec977204df1e41d51a291b62c8e5
```

Comment:
no need for this comment, the test will ensure we are compatible

Reply:
I just wanted to point out why it was done this specific way.